### PR TITLE
doc(cce/addon): update docs of cce addon

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

update docs of cce addon

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes NONE

**Special notes for your reviewer**:

**Release note**:

```release-note
1. update doc of resource cce addon
2. update addons description in example path
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: docs/resources/cce_addon.md [hunk 1]
```diff
@@ -37,29 +37,32 @@ The following arguments are supported:
 
 * The `values` block supports:
 
-* `basic` - (Optional, Map, ForceNew) Key/Value pairs vary depending on the add-on.
+* `basic_json` - (Optional, String, ForceNew) A json string vary depending on the addon.
```
Comment (ShiChangkuo) [inline docs/resources/cce_addon.md:40]: suggest to put all xxx_json together

File: examples/cce/basic/cce-addon-templates.md [hunk 2]
```diff
@@ -30,24 +31,25 @@ data "huaweicloud_cce_addon_template" "autoscaler" {
 }
 
 resource "huaweicloud_cce_addon" "autoscaler" {
-  cluster_id    = var.cluster_id
+  cluster_id = var.cluster_id
   template_name = "autoscaler"
-  version       = "1.19.6"
+  version    = "1.19.6"
```
Comment (ShiChangkuo) [inline examples/cce/basic/cce-addon-templates.md]: some lines arenot aligned with =, please have a check.

File: docs/resources/cce_addon.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/cce_addon.md:40]: fixed

File: examples/cce/basic/cce-addon-templates.md (hunk 2 above)
Comment (Jason-Zhang9309) [reply examples/cce/basic/cce-addon-templates.md]: fixed

**Pull request merged**